src/lsli/config.py

```diff
@@ -0,0 +1,55 @@
+"""
+config.py: Configuration values. Secrets to be handled with Secrets Manager
+"""
+
+import logging
+import socket
+
+SKID_NAME = "lsli_skid"
+
+AGOL_ORG = "https://ddwlead-hub.maps.arcgis.com"
+SENDGRID_SETTINGS = {  #: Settings for SendGridHandler
+    "from_address": "noreply@utah.gov",
+    "to_addresses": ["jdadams@utah.gov", "aamirgol@utah.gov"],
```

Comment:
Thoughts on swapping this to ugrc-developers?

Reply:
I don't see the need to spam you guys with this, and if someone not in the data team (like Zach) ends up taking this over and running it, they'll have to change it anyways.